### DISCUSSION TITLE
po-popover

### DISCUSSION THREAD

**Discussion by duanecorrea** [po-ui/po-angular#947] [Tire suas dúvidas]
Boa tarde,

Existe alguma maneira de eu criar um popover para uma coluna de uma table?

![Capturar](https://user-images.githubusercontent.com/30302444/126823518-021e2d6d-0759-4aed-bb2b-3d8a7e4a4b52.PNG)

Eu tentei criando um cell-template com botões mas não deu certo...

HELP

**Comment by jhonyeduardo** [comment]
Bom dia @duanecorrea 

Consegue criar um exemplo basico neste ambiente? https://stackblitz.com/edit/po-ui
Para avaliarmos o que pode estar acontecendo e tentar ajudar.

**Reply by duanecorrea** [reply]
Bom dia,

Coloquei o exemplo lá.

Obrigado.
